plugins/outputs/http/http_test.go

```diff
@@ -0,0 +1,100 @@
+package http
```

Comment:
how is the code coverage with these tests? in this case we should be able to get up to 100%, and you should probably be implementing tests using the `httptest` package.